src/sagas.js

```diff
@@ -265,7 +274,7 @@ function* fetchVmsByPageVLower (action) {
     if (!shallowFetch) {
       yield fetchConsoleMetadatas({ vms: internalVms })
       yield fetchDisks({ vms: internalVms })
-      yield fetchVmsCDRom({ vms: internalVms })
+      yield fetchVmsCdRom({ vms: internalVms, current: false })
```

Comment:
As I see `fetchVmsCdRom` has parameters `{ vms }`, thus remove, please, `current`.

Reply:
`current: false` for all VMs to make sure that the statically configured ISO is loaded as a base.  

`current: true` is then used when a VM's full details are loaded when entering a details page.

Reply:
`fetchVmsCdRom` was wrong, `current` should be passed here.

`fetchVmsByPageVLower` and `fetchVmsByCountVLower` both need to fetch the `current: false` cd-rom info to match what the >=4.2 fetches pull.